### PR TITLE
Log designation

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: certificate_agent.py [hunk 1]
```diff
@@ -120,11 +120,12 @@ def main():
                     name=name,
                     course_id=course_id,
                     grade=grade,
+                    designation=designation,
```
Comment (stvstnfrd) [inline certificate_agent.py]: We're passing in designation to `format`, but I'm not seeing `designation` anywhere in the formatted string.

File: certificate_agent.py [hunk 2]
```diff
@@ -120,11 +120,12 @@ def main():
...
                 )
             )
             (download_uuid,
              verify_uuid,
-             download_url) = cert.create_and_upload(name, grade=grade)
+             download_url) = cert.create_and_upload(name, grade=grade, designation=designation)
```
Comment (stvstnfrd) [inline certificate_agent.py]: Now we're passing `designation` to `create_and_upload`, but you didn't actually update `create_and_upload` to accept this `kwarg`.

File: gen_cert.py [hunk 3]
```diff
@@ -517,7 +518,7 @@ def _generate_certificate(
         achievements_paragraph = u"{0}{1}".format(achievements_string, achievements_description_string)
 
         # Overide achievements/interstitial strings with yaml designations info based on designation
-        designation = self.designation
+        designation = designation or self.designation
```
Comment (caesar2164) [inline gen_cert.py]: I don't think `self.designation` is being defined here...

File: gen_cert.py [hunk 4]
```diff
@@ -426,6 +426,7 @@ def _generate_certificate(
         verify_dir,
         filename=TARGET_FILENAME,
         grade=None,
+        designation=None,
```
Comment (stvstnfrd) [inline gen_cert.py]: It doesn't look like we updated the call to `_generate_certificate`?

**Review by stvstnfrd — CHANGES_REQUESTED** [review]
This looks incomplete.

File: gen_cert.py (hunk 3 above)
Comment (stvstnfrd) [reply gen_cert.py]: Let's start with just adding the log statements (we should add one here too),
to see what's actually there.
`self.designation` should be set via the constructor.

**Comment by stvstnfrd** [comment]
needs squashed, but +1

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**